hacep/src/test/java/org/kie/hacep/consumer/BidirectionalMapTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.hacep.consumer;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BidirectionalMapTest {
+
+    @Test
+    public void bidirectionalMapTest(){
```

Comment:
Can you please test corner cases in addition to the happy path: what if the key does not exist etc

Reply:
Ok